pkg/client/clientset_generated/internalclientset/typed/apps/internalversion/apps_client.go

```diff
@@ -71,17 +71,12 @@ func New(c rest.Interface) *AppsClient {
 }
 
 func setConfigDefaults(config *rest.Config) error {
```

Comment:
these changes weren't generated?

Reply:
> these changes weren't generated?

They were (otherwise verify fails).

Reply:
ok, saw a "generated" commit split out and thought this commit was limited to manual changes

Reply:
> ok, saw a "generated" commit split out and thought this commit was limited to manual changes

fixed